src/libs/highlevel/README.md

```diff
@@ -165,6 +166,9 @@ The callback must interrupt the thread of execution in some way (e.g. by calling
 The API supports the following types, which are taken from the CORBA specification:
 
 * **String**: a string of characters, represented by `KDB_TYPE_STRING` in metadata
+
+TODO: what does that mean?
```

Comment:
What does "`KDB_TYPE_STRING` in metadata" mean?

Reply:
We could be a bit more clear on that yes. We should add something along the lines of:

The API uses the `type` metakey to distinguish types. For each of the following types the corresponding `KDB_TYPE_*` macros defines the value that Elektra looks for in this metakey.

We should probably also insert the values of the macros into the README, i.e.:

* **String**: a string of characters, represented by `KDB_TYPE_STRING` (`"string"`) in metadata

Reply:
Why would a user need these macros?

Reply:
Most users won't, but if they use `elektraGetType` or `elektraSetRawString` they might want to use `KDB_TYPE_*` (they are actually constants not macros) instead of raw strings to prevent typos and enable IDE code completion.

It would probably suffice to mention once, that a `KDB_TYPE_*` constant exists for each type and then just use the string values in the descriptions.

Reply:
Thank you, can you please do it? (Is now in master)